tests/test_wsgi.py

```diff
@@ -1,14 +1,10 @@
 import threading
 import time
 import unittest
+from urllib.parse import unquote_to_bytes as unquote
```

Comment:
It is not a good idea to import a function with a name that shadows a different function so...
```suggestion
from urllib.parse import unquote_to_bytes
```
And then modify line 66 to match.